src/server/routes/actions.spec.ts

```diff
@@ -35,8 +35,8 @@ jest.mock("../../config", () => {
     config: {
       Get: mockConfig.mockImplementation(
         (key: string) =>
-          // eslint-disable-next-line security/detect-object-injection
-          ({ NETWORK: "rococo" })[key], // minimal viable config on the initial import
+          // eslint-disable-next-line
+          ({ NETWORK: "rococo" }[key]), // minimal viable config on the initial import
```

Comment:
Same here, I don't have the issue if I run `fix` and `lint`.

Reply:
Same as https://github.com/paritytech/polkadot-testnet-faucet/pull/372#discussion_r1587665946